beat/heart.py

```diff
@@ -22,6 +22,7 @@
 from pyrocko.guts_array import Array
 
 from pyrocko import crust2x2, gf, cake, orthodrome, trace, util
+from pyrocko.dataset import ep_crust
```

Comment:
need a try-except surrounding that in case the pyrocko version does not include it, and then set the ep_crust to False or better raise an error with a message that pyrocko needs updating ... Its still not merged there right? Will also look through your PR there, .... soonish.